Тестирование Веб-приложения

vladislav1tv opened the pull request:
Вход
Переход в личный кабинет
Переход из личного кабинета в конструктор
Выход из аккаунта
Раздел «Конструктор»

Добавлен yandexdriver

vladislav1tv committed:
Тестирование Веб-приложения

Вход
Переход в личный кабинет
Переход из личного кабинета в конструктор
Выход из аккаунта
Раздел «Конструктор»

Добавлен yandexdriver

pomegranate282922 commented on src/main/java/WebDriverFactory.java:
```diff
@@ -0,0 +1,23 @@
+            }
+        }
+        return webDriver;
+    }
```
Отлично сделан выбор браузера

pomegranate282922 commented on src/main/java/models/ResponseUser.java:
```diff
@@ -0,0 +1,10 @@
+package models;
+import lombok.Data;
+
+@Data
```
Отлично что используется ломбок

pomegranate282922 commented on src/test/java/RegisterTest.java:
```diff
@@ -0,0 +1,58 @@
+        steps.deleteUser(accessToken).statusCode(202);
+    }
+
+    private String generateRandomEmail() {
```
Этот метод лучше вынести из тестового класса и переиспользовать 
Можно улучшить

pomegranate282922 reviewed (commented):
Работа отличная! Здорово сделан выбор браузера. Применено апи для создания и удаления юзеров. Везде даны описания тестов и степы.
Зачет!

vladislav1tv committed:
Тестирование Веб-приложения

Вход
Переход в личный кабинет
Переход из личного кабинета в конструктор
Выход из аккаунта
Раздел «Конструктор»

Добавлен yandexdriver
